.travis.yml

```diff
@@ -63,3 +65,7 @@ script:
     - go test -c -tags "apparmor seccomp `./btrfs_tag.sh` `./libdm_tag.sh` `./ostree_tag.sh` `./selinux_tag.sh`" ./cmd/buildah
     - tmp=`mktemp -d`; mkdir $tmp/root $tmp/runroot; sudo PATH="$PATH" ./buildah.test -test.v -root $tmp/root -runroot $tmp/runroot -storage-driver vfs -signature-policy `pwd`/tests/policy.json
     - cd tests; sudo PATH="$PATH" ./test_runner.sh
+    - go test -race -coverprofile=coverage.txt -covermode=atomic
```

Comment:
I think you have to pass the -tags line from above.

Reply:
BTW Does this do the tests a second time?  Can you do these in single command?

Reply:
I'm not sure.  The documentation/examples are very sparse.  Will play with it.

Reply:
@rhatdan I believe it doesn't run all tests but `unit-tests`.
I think there is only one `unit-test` which is `TestGetImageName`.

@TomSweeneyRedHat feel free to correct me.

Reply:
I would think you could just add
-race -coverprofile=coverage.txt -covermode=atomic
to the first go test line and it would do everything.

Reply:
Oh! Of course, that's even better.

Reply:
I've merged the two 'go tests' lines per @rhatdan's suggestion